pkg/generator/generator.go

```diff
@@ -285,8 +285,17 @@ func ComputeFingerprint(args ...interface{}) string {
 
 func AddAssetChecks(cfg *config.Config, a config.Target, l log.Logger) error {
 	checks := []config.Check{}
+	var policy config.Policy
 	for ref, ch := range cfg.CheckTypes {
-		if stringInSlice(a.AssetType, ch.Assets) && filterChecktype(ch.Name, cfg.Conf.IncludeR, cfg.Conf.ExcludeR) {
+		if cfg.Conf.Policy != "" {
+			policy = GetPolicy(cfg)
+		}
+		if stringInSlice(a.AssetType, ch.Assets) && filterChecktype(ch.Name, cfg.Conf.IncludeR, cfg.Conf.ExcludeR, policy) {
+			for _, c := range policy.Checks {
+				if ch.Name == string(c.Checktype) {
+					a.Options = mergeOptions(c.Options, a.Options)
```

Comment:
After that you can break the loop

Reply:
I agree, but this code doesn't exist anymore.